composer.json

```diff
@@ -9,7 +9,7 @@
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
         "ext-apcu": "^5.1.10",
-        "laminas/laminas-cache": "^3.0"
+        "laminas/laminas-cache": "^3.12.0"
```

Comment:
Bump needed for transient dependencies causing service manager downgrade. See https://github.com/laminas/laminas-cache-storage-adapter-blackhole/pull/57